include/open62541/server_pubsub.h

```diff
@@ -174,6 +174,7 @@ struct UA_PubSubConnectionConfig {
     size_t connectionPropertiesSize;
     UA_KeyValuePair *connectionProperties;
     UA_Variant connectionTransportSettings;
+    UA_Server *server;
```

Comment:
An alterntive could be to remove this flag here and store it internaly during the UA_Server_addPubSubConnection call.